toolkit/tools/internal/packagerepo/repocloner/rpmrepocloner/rpmrepocloner.go

```diff
@@ -69,60 +92,56 @@ const (
 
 // RpmRepoCloner represents an RPM repository cloner.
 type RpmRepoCloner struct {
-	chroot         *safechroot.Chroot
-	usePreviewRepo bool
-	cloneDir       string
+	chroot                *safechroot.Chroot
+	chrootCloneDir        string
+	defaultMarinerRepoIDs []string
+	externalRepoIDs       map[string]bool
+	mountedCloneDir       string
+	repoIDCache           string
+	reposArgsList         [][]string
```

Comment:
I think we can get away with just `[]string` here.
```suggestion
	reposArgsList         []string
```

Reply:
Keep, `[][]string`